### PR TITLE
build script to build and test for package azure-pipeline-go

### PR DESCRIPTION
**Pull request by vikasgupta8**
build script to build and test for package azure-pipeline-go

### CODE REVIEW

**Comment by seth-priya** [comment]
@vikasgupta8 I tried to execute this script in a clean UBI container and it fails with the following error

[root@b0af0ac2fe70 azure-pipeline-go]# # Ensure go.mod file exists
[root@b0af0ac2fe70 azure-pipeline-go]# go mod init $PACKAGE_PATH
go: /azure-pipeline-go/go.mod already exists

does it work for you?

**Comment by vikasgupta8** [comment]
Yes it has failed. Previously I was getting the code using 'go get' command, but it was failing with junk character in name.
I have updated the script to use 'git clone', hence it failed.
Now I have removed the 'go mod init' command.

File: a/azure-pipeline-go/azure-pipeline-go_ubi.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# ----------------------------------------------------------------------------
+#
+set -e
+
+PACKAGE_NAME=azure-pipeline-go
+PACKAGE_PATH=github.com/Azure/azure-pipeline-go
+PACKAGE_VERSION=${1:-v0.2.2}
+PACKAGE_URL=https://github.com/Azure/azure-pipeline-go
+
+yum install -y git wget tar gcc-c++
```
[the remark this answers is not in the document]
Comment (vikasgupta8) [reply a/azure-pipeline-go/azure-pipeline-go_ubi.sh:25]: Yes it require gcc.

[the remark this answers is not in the document]
Comment (seth-priya) [reply a/azure-pipeline-go/azure-pipeline-go_ubi.sh:25]: ok

**Comment by seth-priya** [comment]
@vikasgupta8 looks like the branch has conflicts, can you please check and update?

**Review by seth-priya — COMMENTED** [review]
Hi @vikasgupta8 I am not sure if the conflicts are resolved correctly, for example don't know why azure-pipeline-go has been added as a "submodule", can you maybe raise a fresh PR for this one?

**Pull request closed without merging**